### ISSUE TITLE
Does consistent of Dynamically Scaled RoPE require sft?

### ISSUE THREAD

**Issue by dengxiaotian123** [NormXU/Consistent-DynamicNTKRoPE#2]
Does consistent of Dynamically Scaled RoPE require sft?

**Comment by NormXU** [comment]
@dengxiaotian123 Yes. From my experiences, SFT the LLM with consistent dynamically scaled RoPE can improve the long context understanding.
